web/ViewModels/PropertyViewModel.cs

```diff
@@ -49,9 +49,9 @@ public class PropertyAddress
 {
     [Required] [StringLength(10000)] public string AddressLine1 { get; set; } = string.Empty;
 
-    [StringLength(10000)] public string? AddressLine2 { get; set; }
+    [StringLength(10000)] public string AddressLine2 { get; set; } = string.Empty;
```

Comment:
Ah sorry I should have been more clear 😅 it's not an issue for fields that we expect can be null to be nullable, so it's alright for optional fields like AddressLine2/3 which are nullable in the database to be nullable here as well 🙂

Reply:
Ah I see! They'll almost all become nullable in BNH-34 either way